x-pack/plugins/security_solution/public/common/components/event_details/insights/translations.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const INSIGHTS = i18n.translate('xpack.securitySolution.alertDetails.overview.insights', {
+  defaultMessage: 'Insights',
+});
+
+export const PROCESS_ANCESTRY = i18n.translate(
+  'xpack.securitySolution.alertDetails.overview.insights.related_alerts_by_process_ancestry',
```

Comment:
random empty space?

Reply:
Prettier moved it to the next line as the line got quite long with that id